dockerfiles/openedx-edxapp/Dockerfile

```diff
@@ -0,0 +1,139 @@
+# Heavily inspired by Tutor from overhang.io
+
+# This file and other files adapted from Tutor are covered by and subject to the AGPL-3
+# as a clause 5 'aggregate'.
+
+# This dockerfile is not expected to work locally. Only executes correctly within the context
+# of its concourse pipeline.
+
+FROM --platform=linux/amd64 python:3.8.12-buster as base
+
+ENV DEBIAN_FRONTEND=noninteractive
+COPY ./apt-pkg-list /root/
+COPY ./pip_package_lists /root/pip_package_lists
+RUN apt update \
+    && apt install -y --no-install-recommends $(cat /root/apt-pkg-list) \
+    && apt autoremove -y \
+    && apt clean \
+    && rm -rf /var/lib/apt/lists/*
+
+FROM base as locales
+ARG OPENEDX_I18N_VERSION="open-release/olive.3"
+RUN cd /tmp \
+    && curl -L -o openedx-i18n.tar.gz https://github.com/openedx/openedx-i18n/archive/$OPENEDX_I18N_VERSION.tar.gz \
+    && tar xzf /tmp/openedx-i18n.tar.gz \
+    && mkdir -p /openedx/locale/contrib \
+    && mv openedx-i18n-*/edx-platform/locale /openedx/locale/contrib \
+    && rm -rf openedx-i18n*
+
+FROM base as code
+# Expected to be run in concourse where the edx-platform code will come from a git resource @ edx_platform
```

Comment:
Another option is to use the git resource as a trigger, but do a `git clone --depth 1` in the image build using the branch/treeish passed in via the `ARG`